src/optimize/babel/helpers.js

```diff
@@ -0,0 +1,54 @@
+// this file is not transpiled in dev
+
+const env = process.env;
+const fromRoot = require('path').resolve.bind(null, __dirname, '../../../');
+
+if (!env.BABEL_CACHE_PATH) {
+  env.BABEL_CACHE_PATH = fromRoot('optimize/.babelcache.json');
+}
+
+if (!env.WEBPACK_BABEL_CACHE_DIR) {
+  env.WEBPACK_BABEL_CACHE_DIR = fromRoot('optimize/.webpack.babelcache');
```

Comment:
this file doesn't seem to be generating, if it's supposed to be can we add it [here](https://github.com/elastic/kibana/blob/c08d9ab0fa5b883b468a20102e1df7527cc73df7/tasks/build/babel_cache.js) to play nice with package permissions